### ISSUE TITLE
TypeError: expected string or bytes-like object

### ISSUE THREAD

**Issue by danjbrendel** [jbarlow83/OCRmyPDF#612]
Hello, sorry if this is a stupid question, fairly new to this. After doing brew install ocrmypdf, I tried to run from Mac z-shell, but I get the error below. Any help much appreciated!

```
$ ocrmypdf --force-ocr Ritter_102318.pdf Ritter_redo.pdf
Scanning contents: 100%|████████████████████████| 7/7 [00:00<00:00, 12.96page/s]
Start processing 4 pages concurrently
    7 [tesseract] lots of diacritics - possibly poor OCR
OCR: 100%|██████████████████████████████████| 7.0/7.0 [00:16<00:00,  2.33s/page]
An exception occurred while executing the pipeline
Traceback (most recent call last):
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/ocrmypdf/_sync.py", line 367, in run_pipeline
    exec_concurrent(context)
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/ocrmypdf/_sync.py", line 301, in exec_concurrent
    pdf = post_process(pdf, context)
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/ocrmypdf/_sync.py", line 241, in post_process
    pdf_out = metadata_fixup(pdf_out, context)
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/ocrmypdf/_pipeline.py", line 768, in metadata_fixup
    del meta['dc:title']
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/pikepdf/models/metadata.py", line 391, in __exit__
    self._apply_changes()
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/pikepdf/models/metadata.py", line 463, in _apply_changes
    self._update_docinfo()
  File "/usr/local/Cellar/ocrmypdf/10.3.3/libexec/lib/python3.8/site-packages/pikepdf/models/metadata.py", line 431, in _update_docinfo
    value = re_xml_illegal_chars.sub('', value)
TypeError: expected string or bytes-like object
```

**Comment by jbarlow83** [comment]
Any chance you can send that file? It looks like there's something unusual about its metadata block that causes an error.

**Comment by danjbrendel** [comment]
Sure thanks! Though I get the same error message for some other files, too.
Find two of them attached.

**Comment by danjbrendel** [comment]
FYI here's another that threw the same error.

Thanks very much for taking the time to check this issue out. I'm a local
newspaper reporter, and I've written some code to churn through campaign
finance documents for local elections and to extract and compile tabular
campaign contribution data. At these lower levels, the info is publicly
available, but only as PDFs, which are maintained separately by individual
city clerks. I've not found any central repository where the data is
compiled in a usable format for looking at long-term trends and whatnot. So
I figured I'd try to do that myself! At least for my region. But the OCR
that's done on the clerks' docs isn't always very good, which is why I'd
love to include your "redo" feature in my loop, before the tabular
extraction module does its thing. More info than you wanted, probably, I
just wanted to say it's a cool program you've made and to say thanks for
checking out this issue.

**Comment by jbarlow83** [comment]
Unfortunately GitHub drops attachments that are emailed to it so I can't see what you sent. You have to log in and attach them to a comment.

Camelot and Tabula are tools for extracting tabular data from PDF, if you haven't found them already. When OCR is involved they can be hit and miss or need a lot of manual labor to fix.

**Comment by danjbrendel** [comment]
Oops sorry. Here are attachments of some examples where I ran into this issue.
[Blakespear_2020.pdf](https://github.com/jbarlow83/OCRmyPDF/files/5077351/Blakespear_2020.pdf)
[2000_Statements_Bond.pdf](https://github.com/jbarlow83/OCRmyPDF/files/5077353/2000_Statements_Bond.pdf)
[Ritter_102318.pdf](https://github.com/jbarlow83/OCRmyPDF/files/5077354/Ritter_102318.pdf)

**Comment by jbarlow83** [comment]
Fixed for v11.0.2
